### DISCUSSION TITLE
Test Android App

### DISCUSSION THREAD

**Discussion by Backxtar** [TeKrop/overfast-api#49] [General]
You can test the Android App if you want :)
It’s not the final Release but good enough to test it :)

https://github.com/Backxtar/watch-over-support/releases/tag/v1.0.0

This is btw only the support git :) I can’t release all sourcecode because of licensing :3

**Comment by TeKrop** [comment]
Backxtar's app is now available on Play Store, as stated in the readme of the project : https://play.google.com/store/apps/details?id=de.backxtar.watchoveroverwatch

**Reply by Backxtar** [reply]
Nice Ty :) 
Btw the new gamemode flashpoint is missing :) 

Today there will be an update.
